Server/furnace.txt

```diff
@@ -43,73 +44,73 @@
-GrayTerracotta      = GrayGlazedTerracotta               $ 0.1
-GreenTerracotta     = GreenGlazedTerracotta              $ 0.1
-IronOre             = IronIngot                          $ 0.7
-IronAxe             = IronNugget                         $ 0.1
-IronBoots           = IronNugget                         $ 0.1
-IronChestplate      = IronNugget                         $ 0.1
-IronHorseArmor      = IronNugget                         $ 0.1
-IronHelmet          = IronNugget                         $ 0.1
-IronHoe             = IronNugget                         $ 0.1
-IronLeggings        = IronNugget                         $ 0.1
-IronPickaxe         = IronNugget                         $ 0.1
-IronShovel          = IronNugget                         $ 0.1
-IronSword           = IronNugget                         $ 0.1
-LapisOre            = LapisLazuli                        $ 0.2
-LightBlueTerracotta = LightBlueGlazedTerracotta          $ 0.1
-LightGrayTerracotta = LightGrayGlazedTerracotta          $ 0.1
-LimeTerracotta      = LimeGlazedTerracotta               $ 0.1
-Log                 = CharCoal                           $ 0.15
-Log2                = CharCoal                           $ 0.15
-MagentaTerracotta   = MagentaGlazedTerracotta            $ 0.1
-Mutton              = CookedMutton                       $ 0.35
-NetherQuartzOre     = NetherQuartz                       $ 0.2
-Netherrack          = NetherBrick                        $ 0.1
-OrangeTerracotta    = OrangeGlazedTerracotta             $ 0.1
-PinkTerracotta      = PinkGlazedTerracotta               $ 0.1
-Porkchop            = CookedPorkchop                     $ 0.35
-Potato              = BakedPotato                        $ 0.35
-PurpleTerracotta    = PurpleGlazedTerracotta             $ 0.1
-Rabbit              = CookedRabbit                       $ 0.35
-RedTerracotta       = RedGlazedTerracotta                $ 0.1
-RedstoneOre         = Redstone                           $ 0.7
-Salmon              = CookedSalmon                       $ 0.35
-Sand                = Glass                              $ 0.1
-StoneBrick          = CrackedStoneBricks                 $ 0.1
-WetSponge           = Sponge                             $ 0.15
-WhiteTerracotta     = WhiteGlazedTerracotta              $ 0.1
-YellowTerracotta    = YellowGlazedTerracotta             $ 0.1
+Beef                   = Steak                              $ 0.35
+BlackTerracotta        = BlackGlazedTerracotta              $ 0.1
+BlueTerracotta         = BlueGlazedTerracotta               $ 0.1
+BrownTerracotta        = BrownGlazedTerracotta              $ 0.10
+Cactus                 = CactusGreen                        $ 0.2
+ChainmailBoots^-1      = IronNugget                         $ 0.1
+ChainmailChestplate^-1 = IronNugget                         $ 0.1
+ChainmailHelmet^-1     = IronNugget                         $ 0.1
+ChainmailLeggings^-1   = IronNugget                         $ 0.1
```

Comment:
The spec at the top says `ItemType : Damage`, here you use `ItemType ^ Damage`.

Reply:
Changed it to `:-1`